src/hooks/useSafeTransactionStatus.ts

```diff
@@ -0,0 +1,53 @@
+import { defineMessages } from 'react-intl';
+
+import { ETHEREUM_NETWORK } from '~constants';
+import { useGetSafeTransactionStatusQuery } from '~gql';
+import { ColonyAction } from '~types';
+
+export enum TRANSACTION_STATUS {
+  COMPLETED = 'Completed',
+  ACTION_NEEDED = 'Action neeeded',
+}
+
+const displayName = 'utils.safe';
+
+export const MSG = defineMessages({
+  [TRANSACTION_STATUS.COMPLETED]: {
+    id: `${displayName}.${[TRANSACTION_STATUS.COMPLETED]}`,
+    defaultMessage: 'Completed',
+  },
+  [TRANSACTION_STATUS.ACTION_NEEDED]: {
+    id: `${displayName}.${[TRANSACTION_STATUS.ACTION_NEEDED]}`,
+    defaultMessage: 'Action needed',
```

Comment:
Would be nice to use the value of the enums here to save some repetition. Whadda you reckon?